src/app/LabelLayer.tsx

```diff
@@ -0,0 +1,197 @@
+/** @jsxImportSource @emotion/react */
+
+// This file contains code related to the label step. Earlier, we had
```

Comment:
This duplication is great; we can factor things together as we learn more, but keeping these separate for now makes a lot of sense to me.

Reply:
Super happy that we are on the same page :D